resources/src/main/java/org/robolectric/res/ResourceTableFactory.java

```diff
@@ -3,46 +3,51 @@
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
 import org.robolectric.util.Logger;
+import org.robolectric.util.PerfStatsCollector;
 
 public class ResourceTableFactory {
   /**
    * Builds an Android framework resource table in the "android" package space.
    */
   public PackageResourceTable newFrameworkResourceTable(ResourcePath resourcePath) {
-    PackageResourceTable resourceTable = new PackageResourceTable("android");
+    return PerfStatsCollector.getInstance().measure("load legacy framework resources", () -> {
+      PackageResourceTable resourceTable = new PackageResourceTable("android");
 
-    if (resourcePath.getRClass() != null) {
-      addRClassValues(resourceTable, resourcePath.getRClass());
-      addMissingStyleableAttributes(resourceTable, resourcePath.getRClass());
-    }
-    if (resourcePath.getInternalRClass() != null) {
-      addRClassValues(resourceTable, resourcePath.getInternalRClass());
-      addMissingStyleableAttributes(resourceTable, resourcePath.getInternalRClass());
-    }
+      if (resourcePath.getRClass() != null) {
```

Comment:
It's default IntelliJ IDEA/Android Studio formatting or your custom one?

Reply:
I'll reformat.